src/terraboot/infra.clj

```diff
@@ -5,28 +5,58 @@
 
 (def infra-path "infra/")
 
-;; other todo: public slave port and listeners being parameters (as this is determined by applications)
 (defn generate-json [target]
   (let [account-number "165664414043"
-        azs [:a]]
+        azs [:a]
+        mesos-ami "ami-cfca25a0"
+        default-ami "ami-9b9c86f7"]
     (condp = target
       "vpc"     (to-file (vpc-vpn-infra {:vpc-name vpc-name
                                          :account-number account-number
                                          :azs azs
+                                         :default-ami default-ami
                                          :subnet-cidr-blocks {:a {:public "172.20.0.0/24"
                                                                   :private "172.20.8.0/24"}}}) "vpc/vpc.tf")
       "staging" (to-file (cluster-infra {:vpc-name vpc-name
+                                         :account-number account-number
                                          :cluster-name "staging"
                                          :min-number-of-masters 3
                                          :max-number-of-masters 3
                                          :master-disk-allocation 20
+                                         :master-instance-type "m4.large"
                                          :min-number-of-slaves 2
                                          :max-number-of-slaves 2
+                                         :slave-instance-type "m4.xlarge"
                                          :min-number-of-public-slaves 1
                                          :max-number-of-public-slaves 1
+                                         :public-slave-instance-type "t2.medium"
+                                         :public-slave-elb-listeners [{:lb-port 443 :lb-protocol "https" :port 80 :protocol "http" :cert-name "c512707d-bbec-4859-ab22-0f5fbad62a22"}
```

Comment:
this looks like the place for all of our specific config, so I'm guessing this UUID is fine?